Translation corrections (#456)
07th-mod/higurashi-console-arcs: Update/ura_016.txt

//rせいぜい死の恐怖に怯えさせてから撃ち殺して溜飲を下げ、他の捜査員が来る前に滑走路を横切ってターミナルに入り、居並ぶ一般客たちを盾にして脱出を図るつもりだったのだ。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "せいぜい死の恐怖に怯えさせてから撃ち殺して溜飲を下げ、他の捜査員が来る前に滑走路を横切ってターミナルに入り、居並ぶ一般客たちを盾にして脱出を図るつもりだったのだ。",

> 			NULL, "He'd been planning to terrify her with the fear of death before shooting her to death to satisfy himself, then run across the runway into the terminal before the other investigators arrive, and use the crowd as a shield to escape.", Line_Normal);

Reviewer: ```suggestion
			NULL, "He'd been planning to terrify her with the fear of impending death before killing her to satisfy himself. Then he'd run across the runway into the terminal before the other investigators arrive, and use the crowd as a shield to escape.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#646571>花田</color>", NULL, "<color=#646571>Hanada</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 32, "ps3/s14/32/vt4bb_hanada1215", 256, TRUE);
	OutputLine(NULL, "「もう殺す…殺す殺す殺す殺す、殺すッッ！！",

> 			NULL, "\"I'll kill you... I'll kill you, I'll kill you, I'll kill you, I'll kill you, I'll kill you!!", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"I'll kill you... I'll kill you, I'll kill you, I'll kill you!!", Line_WaitForInput);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#646571>花田</color>", NULL, "<color=#646571>Hanada</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 32, "ps3/s14/32/vt4bb_hanada1217", 256, TRUE);
	OutputLine(NULL, "「これまであんたが積み上げてきた努力も、運も、才能も全部ぶち壊しにして！",

> 			NULL, "\"I'll destroy all the effort, luck, and talent you've been building up to reach this point in an instant!", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"All the effort, luck, and talent you've been building up to reach this point will be destroyed in an instant!", Line_WaitForInput);
```

			NULL, "\"Stop iiiit!!", Line_WaitForInput);
	ModPlayVoiceLS(4, 32, "ps3/s14/32/vt4bb_hanada1226", 256, TRUE);
	OutputLine(NULL, "　もう、やめてくれぇぇぇっっ！！！」",

> 			NULL, " Just stop it alreadyyyyyy!!!\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " That's enooooough!!!\"", Line_Normal);
```

			NULL, "\"Every single person betrayed me and disgraced me!", Line_WaitForInput);
	ModPlayVoiceLS(4, 32, "ps3/s14/32/vt4bb_hanada1241", 256, TRUE);
	OutputLine(NULL, "　何もかもが信用できなくて、憎んで、憎まれて！",

> 			NULL, " I could no longer trust anything, so I hated and was hated!", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, " I could no longer trust anything, so I hated them and was hated in return!", Line_WaitForInput);
```

//rいや、…本当は頭では、わかっている。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "いや、…本当は頭では、わかっている。",

> 			NULL, "No... In actuality, Hanada knew.", Line_Normal);

Reviewer: ```suggestion
			NULL, "No... The truth is, Hanada knew.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f5e6d3>安藤</color>", NULL, "<color=#f5e6d3>Andou</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 0, "ps3/s14/00/vt4bb_ando1026", 256, TRUE);
	OutputLine(NULL, "「無茶しやがって…この野郎！！」",

> 			NULL, "\"You dumbass... Ithat was beyond reckless!!\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"You dumbass... That was beyond reckless!!\"", Line_Normal);
```
